modules/configuring-administration-events-report-retention-settings.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// *operating/using-the-administration-events-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-administration-events-report-retention-settings_{context}"]
+= Configuring administration events report retention settings
+
+By configuring the retention settings, you can determine when the administration events report expires.
+
+.Procedure
+
+. In the {product-title-short} portal, navigate to *Platform Configuration ->  System Configuration*. You can configure the following setting for administration events:
+
+* *Administration events retention days*: The number of days to keep your administration events report. By specifying the number of days, you can control when these reports expire. This is important for managing your administration events report and making sure that you retain the information for the desired duration. By default, the administration events report is retained for 4 days.
```

Comment:
More of a nit, but I'd love for us to highlight the default retention period of 4 days more.

Also, it's probably worth to mention that not the _complete_ report expires but rather only single events within that dashboard (there's no concept of an "administration event report" so probably we shouldn't use that verbiage to begin with).

My fear with using "administration events report" is that customers will think of the reporting feature, and expect a scheduled report of their current administration events (i.e. found issues within their system) via e.g. E-Mail, which isn't supported at all.